### PR TITLE
Show EgressCIDRs in "oc get hostsubnets"

### PR DESCRIPTION
**Pull request by danwinship**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1609149

### CODE REVIEW

File: pkg/printers/internalversion/printer.go [hunk 1]
```diff
@@ -987,7 +987,7 @@ func printGroupList(list *userapi.GroupList, w io.Writer, opts kprinters.PrintOp
 
 func printHostSubnet(h *networkapi.HostSubnet, w io.Writer, opts kprinters.PrintOptions) error {
 	name := formatResourceName(opts.Kind, h.Name, opts.WithKind)
-	_, err := fmt.Fprintf(w, "%s\t%s\t%s\t%s\t[%s]\n", name, h.Host, h.HostIP, h.Subnet, strings.Join(h.EgressIPs, ", "))
+	_, err := fmt.Fprintf(w, "%s\t%s\t%s\t%s\t[%s]\t[%s]\n", name, h.Host, h.HostIP, h.Subnet, strings.Join(h.EgressCIDRs, ", "), strings.Join(h.EgressIPs, ", "))
 	return err
```
Comment (pravisankar) [inline pkg/printers/internalversion/printer.go]: We also need to update the host subnet describer: https://github.com/openshift/origin/blob/master/pkg/oc/lib/describe/describer.go#L1755

Comment (danwinship) [reply pkg/printers/internalversion/printer.go]: @pravisankar righto. fixed

**Comment by knobunc** [comment]
@bparees will you please take a look at this since we need approval for your area.

**Pull request merged**